README.md

```diff
@@ -15,3 +15,9 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+**Since my implementation just iterates through every element in an adjacency matrix until it finds a cycle, its worst case runtime complexity would be $\theta(|V|^2)$  This is because, since it starts searching at the first edge in the first vertes, it will have to iterate from the very start of the matrix, enocunter nothing, and keep going until it reaches the very last edge to check, which would be graph[|V|][|V|]. Since it has to iterate through every single possible edge, this means it has to iterate through $|V|^2$ times, which ends up being the final rutnime complexity.**
```

Comment:
Your justification talks about edges, but the number of edges isn't part of the complexity?